config/routes.rb

```diff
@@ -89,6 +89,7 @@
   get '/popular/recent' => 'likes#recent'
 
   get '/upload' => 'spectrums#new'
+  post '/spectrums/create' => 'spectrums#create'
```

Comment:
How was 'create' being routed to before? There was no route to spectrums#create before

Reply:
It was possibly being generated by the deprecated way of generic routing. Rails dropped it since it had security issues